src/core/image.js

```diff
@@ -260,12 +319,14 @@ var PDFImage = (function PDFImageClosure() {
   };
 
   PDFImage.createMask = function({ imgArray, width, height,
-                                   imageIsFromDecodeStream, inverseDecode, }) {
+                                   imageIsFromDecodeStream, inverseDecode,
+                                   resizeHeight, resizeWidth, }) {
```

Comment:
It looks a bit better to reverse width and height here, just like in the rest of the patch, i.e., first width, then height.

Reply:
Done.